platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/ConsensusHashManager.java

```diff
@@ -94,7 +96,8 @@ public ConsensusHashManager(
             final DispatchBuilder dispatchBuilder,
             final AddressBook addressBook,
             final ConsensusConfig consensusConfig,
-            final StateConfig stateConfig) {
+            final StateConfig stateConfig,
+            final Hash currentEpochHash) {
```

Comment:
missing param doc for `currentEpochHash`

Reply:
added